pagerduty/event_orchestration_cache_variable_test.go

```diff
@@ -0,0 +1,711 @@
+			},
+		},
+		Configuration: &EventOrchestrationCacheVariableConfiguration{
+			Type:       "trigger_event_count",
+			TTLSeconds: 70,
+		},
+		CreatedAt: "2024-02-12T14:44:58Z",
+		CreatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR7",
+			Self: "https://api.pagerduty.com/users/P8B9WR7",
+			Type: "user_reference",
+		},
+		UpdatedAt: "2024-02-13T12:41:43Z",
+		UpdatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR8",
+			Self: "https://api.pagerduty.com/users/P8B9WR8",
+			Type: "user_reference",
+		},
+	}
+
+	if !reflect.DeepEqual(resp, want) {
+		t.Errorf("returned \n\n%#v want \n\n%#v", resp, want)
+	}
+}
+
+func TestServiceOrchestrationICacheVariableUpdate(t *testing.T) {
+	setup()
+	defer teardown()
+	oId := "31112383-c14b-4464-b8fc-660f2508f43b"
+	oType := "service"
+	id := "460d9629-269a-4d26-8b6b-b96d61102436"
+	input := &EventOrchestrationCacheVariable{
+		Disabled: true,
+	}
+	url := fmt.Sprintf("%s/services/%s/cache_variables/%s", eventOrchestrationBaseUrl, oId, id)
+
+	mux.HandleFunc(url, func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "PUT")
+		v := new(EventOrchestrationCacheVariablePayload)
+		json.NewDecoder(r.Body).Decode(v)
+		if !reflect.DeepEqual(v.CacheVariable, input) {
+			t.Errorf("Request body = %+v, want %+v", v, input)
+		}
+		w.Write([]byte(`{
+			"cache_variable": {
+				"id": "460d9629-269a-4d26-8b6b-b96d61102436",
+				"name": "updated_example",
+				"configuration": {
+					"type": "trigger_event_count",
+					"ttl_seconds": 10
+				},
+				"disabled": true,
+				"created_at": "2024-02-12T14:44:58Z",
+				"created_by": {
+					"id": "P8B9WR7",
+					"self": "https://api.pagerduty.com/users/P8B9WR7",
+					"type": "user_reference"
+				},
+				"updated_at": "2024-02-13T12:41:43Z",
+				"updated_by": {
+					"id": "P8B9WR8",
+					"self": "https://api.pagerduty.com/users/P8B9WR8",
+					"type": "user_reference"
+				}
+			}
+		}`))
+	})
+
+	resp, _, err := client.EventOrchestrationCacheVariables.UpdateContext(context.Background(), oType, oId, id, input)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	want := &EventOrchestrationCacheVariable{
+		ID:   "460d9629-269a-4d26-8b6b-b96d61102436",
+		Name: "updated_example",
+		Configuration: &EventOrchestrationCacheVariableConfiguration{
+			Type:       "trigger_event_count",
+			TTLSeconds: 10,
+		},
+		Disabled: true,
+		CreatedAt: "2024-02-12T14:44:58Z",
+		CreatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR7",
+			Self: "https://api.pagerduty.com/users/P8B9WR7",
+			Type: "user_reference",
+		},
+		UpdatedAt: "2024-02-13T12:41:43Z",
+		UpdatedBy: &EventOrchestrationCacheVariableReference{
+			ID:   "P8B9WR8",
+			Self: "https://api.pagerduty.com/users/P8B9WR8",
+			Type: "user_reference",
+		},
+	}
+
+	if !reflect.DeepEqual(resp, want) {
+		t.Errorf("returned \n\n%#v want \n\n%#v", resp, want)
+	}
+}
+
+func TestServiceOrchestrationIntegrationDelete(t *testing.T) {
```

Comment:
Typo in `TestServiceOrchestrationIntegrationDelete`
```suggestion
func TestServiceOrchestrationCacheVariableDelete(t *testing.T) {
```